.github/workflows/nomad.yaml

```
@@ -55,15 +55,15 @@ jobs:
 
           - scenario-name: write_validated_must_get_agent_activity
             required-nodes: 2
-
+    env:
+      RUN_ID: "${{ matrix.job-name || matrix.scenario-name }}_${{ github.run_id }}"
```

actionlint (1.7.8)

property "job-name" is not defined in object type {required-nodes: number; scenario-name: string}

(expression)